### PR TITLE
DatePicker common component for HFI Calc & FireBat

### PR DESCRIPTION
**Pull request by andrea-williams**
https://wps-pr-1356.apps.silver.devops.gov.bc.ca/hfi-calculator

### CODE REVIEW

File: web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx [hunk 1]
```diff
@@ -69,14 +74,18 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
...
+          </FormControl>
           <DailyViewTable
-            title="HFI Calculator Daily View"
             testId="hfi-calc-daily-table"
             fireCentres={fireCentres}
             dailiesMap={dailiesMap}
-            currentDay={currentDay.toLocaleString()}
-            previousDay={previousDay}
-            nextDay={nextDay}
+            currentDay={dateOfInterest}
```
Comment (conbrad) [inline web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx:88]: Is this used in `DailyViewTable` anymore?

Comment (andrea-williams) [reply web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx]: nope! good catch. I've deleted it now

**Review by conbrad — APPROVED** [review]

**Review by Sybrand — CHANGES_REQUESTED** [review]
- heading should be one line ("Date of Interest (PST-08:00)" takes up two lines.
- re-load of screen happens when I click on date, and click away without changing the date.
- re-load of screen doesn't happen when I change date (only happens after I click away)

**Review by tessmgood — APPROVED** [review]
Noice!

**Review by Sybrand — APPROVED** [review]

**Pull request merged**